kinder/cmd/kinder/get/get.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package get implements the `get` command
+package get
```

Comment:
we add get.go for artifacts, but does it makes sense to have the other subcommands too?
(the build of this file might fail because king/get/* packages are missing).

Reply:
having the kinder list of commands implemented as a superset of the kind list of commands IMO is nice because it avoids users to switch back and forth from the two commands.

the risk is that something will break when we will update the kind dependency, but this applies to all the "kind as a library" surface as well because it is something not yet stable and well defined, so I'm fine with this until kinder is experimental and we can change if required

Reply:
actually, i got confused that we are using go modules and didn't know where the "get" sub-packages are coming from if we are not vendoring the kind cmd package.

but your comment makes sense.
given upstream is aiming for a stable CLI soon, i think that such extensions will be safe, for now.

/lgtm